src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas1/Cas1SpaceBookingTest.kt

```diff
@@ -971,6 +972,93 @@ class Cas1SpaceBookingTest {
         .isOk
       domainEventAsserter.assertDomainEventOfTypeStored(spaceBooking.application!!.id, DomainEventType.APPROVED_PREMISES_PERSON_ARRIVED)
     }
+
+    @Test
+    fun `Recording arrival returns OK when arrival already registered and arrival date matches`() {
```

Comment:
up to you if you want to keep it, but i think it's probably overkill to have a unit and integration test as this  change hasn't introduced any database / SQL / integration changes? In this case i'd keep the unit test for the sake of execution speed.